pallets/liquid-staking/src/bridge.rs

```diff
@@ -108,4 +110,112 @@ where
         }
         Ok(())
     }
+
+    /// unbond on relaychain via xcm.transact
+    pub(crate) fn unbond(value: Balance) -> DispatchResult {
+        let call = StakingUnbondCall::<T> {
+            call_index: [6, 2],
+            value,
+        };
+
+        let msg = WithdrawAsset {
+            assets: vec![MultiAsset::ConcreteFungible {
+                id: MultiLocation::Null,
+                amount: 1_000_000_000_000,
+            }],
+            effects: vec![BuyExecution {
+                fees: MultiAsset::All,
+                weight: 800_000_000,
+                debt: 600_000_000,
+                halt_on_error: true,
+                xcm: vec![Transact {
+                    origin_type: OriginKind::SovereignAccount,
+                    require_weight_at_most: 1_000_000_000,
+                    call: call.encode().into(),
+                }],
+            }],
+        };
+
+        match T::XcmSender::send_xcm(MultiLocation::X1(Junction::Parent), msg) {
+            Ok(()) => {
+                Self::deposit_event(Event::<T>::UnbondCallSent(value));
+            }
+            Err(_e) => {
+                return Err(Error::<T>::UnbondCallFailed.into());
+            }
+        }
+        Ok(())
+    }
+
+    /// rebond on relaychain via xcm.transact
+    pub(crate) fn rebond(value: Balance) -> DispatchResult {
+        let call = StakingRebondCall::<T> {
+            call_index: [6, 19],
+            value,
+        };
+
+        let msg = WithdrawAsset {
+            assets: vec![MultiAsset::ConcreteFungible {
+                id: MultiLocation::Null,
+                amount: 1_000_000_000_000,
+            }],
+            effects: vec![BuyExecution {
+                fees: MultiAsset::All,
+                weight: 800_000_000,
+                debt: 600_000_000,
+                halt_on_error: true,
+                xcm: vec![Transact {
+                    origin_type: OriginKind::SovereignAccount,
+                    require_weight_at_most: 1_000_000_000,
+                    call: call.encode().into(),
+                }],
+            }],
+        };
+
+        match T::XcmSender::send_xcm(MultiLocation::X1(Junction::Parent), msg) {
+            Ok(()) => {
+                Self::deposit_event(Event::<T>::RebondCallSent(value));
+            }
+            Err(_e) => {
+                return Err(Error::<T>::RebondCallFailed.into());
+            }
+        }
+        Ok(())
+    }
+
+    /// withdraw unbonded on relaychain via xcm.transact
+    pub(crate) fn withdraw_unbonded(num_slashing_spans: u32) -> DispatchResult {
```

Comment:
It seems like no test for this method, are we going to add a unit test for it?

Reply:
yes I'm adding it